.github/workflows/_component_prechecks.yml

```diff
@@ -0,0 +1,46 @@
+name: nucliadb Client (py)
```

Comment:
change to `${{ inputs.component }} pre-checks`

Reply:
Oops!